<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1211</width>
    <height>1158</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <widget class="QPushButton" name="SettingsSaveButton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>10</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>SaveSettings</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SettingsLoadButton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>50</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>LoadSettings</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ScanDevButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>80</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Scan
Devices</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ConnectButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>10</y>
      <width>80</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QPushButton" name="RUNButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>210</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {background-color:  rgb(0, 255, 0)}</string>
    </property>
    <property name="text">
     <string>RUN</string>
    </property>
   </widget>
   <widget class="QPushButton" name="STOPButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>210</y>
      <width>80</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {color: white; background-color: gray;}</string>
    </property>
    <property name="text">
     <string>STOP</string>
    </property>
   </widget>
   <widget class="QPushButton" name="RECButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>210</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {background-color:  rgb(200, 0, 0)}</string>
    </property>
    <property name="text">
     <string>REC</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>100</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Frame rate</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="FrameRateSpin">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>120</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="suffix">
     <string> fps</string>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>200</number>
    </property>
    <property name="value">
     <number>10</number>
    </property>
   </widget>
   <widget class="QTextEdit" name="Devices_textEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>161</width>
      <height>131</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="AutoExposeButton">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>40</y>
      <width>111</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>AutoExpose</string>
    </property>
   </widget>
   <widget class="QPushButton" name="AutoGainButton">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>70</y>
      <width>111</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>AutoGain</string>
    </property>
   </widget>
   <widget class="QPushButton" name="WhiteBalanceButton">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>100</y>
      <width>111</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>WhiteBalance</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="All_cams_checkBox">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>10</y>
      <width>75</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>all cams</string>
    </property>
   </widget>
   <widget class="QPushButton" name="FlipXButton">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>140</y>
      <width>111</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Flip X</string>
    </property>
   </widget>
   <widget class="QPushButton" name="FlipYButton">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>170</y>
      <width>111</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Flip Y</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ShowSingleCamButton">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>210</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {background-color:  rgb(0, 200, 0)}</string>
    </property>
    <property name="text">
     <string>Show Cam</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="HWTrig_checkBox">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>260</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>HW-triggered</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="DisableViz_checkBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>300</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Disable Visualization</string>
    </property>
   </widget>
   <widget class="MultiCameraViewer" name="MultiViewWidget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>1191</width>
      <height>781</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
   </widget>
   <widget class="QPushButton" name="ExtraWindowButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>293</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Show in Extra window</string>
    </property>
   </widget>
   <widget class="CameraSettingsTab" name="CameraSettings2" native="true">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>0</y>
      <width>301</width>
      <height>331</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
   </widget>
   <widget class="QPushButton" name="Save_pathButton">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>10</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Choose
Savepath</string>
    </property>
   </widget>
   <widget class="QPushButton" name="REC_calib_Button">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>210</y>
      <width>80</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>REC
Calibration</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="crf_spinBox">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>70</y>
      <width>71</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QComboBox" name="Codec_comboBox">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>40</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>40</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Codec</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>70</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>crf</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>10</y>
      <width>211</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Video Encoding params</string>
    </property>
   </widget>
   <widget class="QPushButton" name="RemoteModeButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>100</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {background-color:  rgb(0, 125, 125)}</string>
    </property>
    <property name="text">
     <string>Enable
REMOTE mode</string>
    </property>
   </widget>
   <widget class="QLabel" name="Client_label">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>150</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>disconnected</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="SessionIDlineEdit">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>180</y>
      <width>371</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>160</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Video name prefix:</string>
    </property>
   </widget>
   <zorder>SettingsSaveButton</zorder>
   <zorder>SettingsLoadButton</zorder>
   <zorder>ScanDevButton</zorder>
   <zorder>ConnectButton</zorder>
   <zorder>RUNButton</zorder>
   <zorder>STOPButton</zorder>
   <zorder>RECButton</zorder>
   <zorder>label_5</zorder>
   <zorder>FrameRateSpin</zorder>
   <zorder>Devices_textEdit</zorder>
   <zorder>AutoExposeButton</zorder>
   <zorder>AutoGainButton</zorder>
   <zorder>WhiteBalanceButton</zorder>
   <zorder>All_cams_checkBox</zorder>
   <zorder>FlipXButton</zorder>
   <zorder>FlipYButton</zorder>
   <zorder>ShowSingleCamButton</zorder>
   <zorder>HWTrig_checkBox</zorder>
   <zorder>DisableViz_checkBox</zorder>
   <zorder>ExtraWindowButton</zorder>
   <zorder>MultiViewWidget</zorder>
   <zorder>CameraSettings2</zorder>
   <zorder>Save_pathButton</zorder>
   <zorder>REC_calib_Button</zorder>
   <zorder>crf_spinBox</zorder>
   <zorder>Codec_comboBox</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>label_3</zorder>
   <zorder>RemoteModeButton</zorder>
   <zorder>Client_label</zorder>
   <zorder>SessionIDlineEdit</zorder>
   <zorder>label_4</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionsaveSettings">
   <property name="text">
    <string>SaveSettings</string>
   </property>
  </action>
  <action name="actionLoadSettings">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>LoadSettings</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MultiCameraViewer</class>
   <extends>QWidget</extends>
   <header location="global">ImageViewer</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CameraSettingsTab</class>
   <extends>QWidget</extends>
   <header>ImageViewer</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
